tests/test_server.py

```diff
@@ -334,6 +330,54 @@ def test_train_internal_error(rasa_app: SanicTestClient):
     assert response.status == 500
 
 
+@pytest.fixture
+def formbot_data():
+    return dict(
+        domain="examples/formbot/domain.yml",
+        config="examples/formbot/config.yml",
+        stories="examples/formbot/data/stories.md",
+        nlu="examples/formbot/data/nlu.md",
+    )
+
+
+def test_train_status(rasa_server, formbot_data):
+    with ExitStack() as stack:
+        payload = {
+            key: stack.enter_context(open(path)).read()
+            for key, path in formbot_data.items()
+        }
+
+    from multiprocessing import Process, Manager
+
+    def train(results):
+        client1 = SanicTestClient(rasa_server, port=PORT)
+        _, train_resp = client1.post("/model/train", json=payload)
+        results["train_response_code"] = train_resp.status
+
+    def train_status(results):
+        # Wait for 1 second for the training to definitively begin
+        time.sleep(1)
```

Comment:
The other option would be to query if a few times in a loop like this:
```python
for i in range(5):
	time.sleep(0.2)
	_, status_resp = client2.get("/model/train/status")
	results["train_status_response_code"] = status_resp.status
	results["train_status_n_jobs"] = status_resp.json["n_jobs"]
	if results["train_status_n_jobs"] == 1:
		break
```

Reply:
yes this should be repeated multiple tries